src/backend/package.json

```diff
@@ -13,8 +13,8 @@
   "author": "Digital Democracy",
   "license": "MIT",
   "dependencies": {
-    "@comapeo/core": "2.0.0",
-    "@comapeo/ipc": "2.0.0",
+    "@comapeo/core": "2.0.1",
+    "@comapeo/ipc": "2.0.2",
```

Comment:
just highlighting that we need this update to core to fix an issue that happens when removing a custom map.